lib/fog/vsphere/models/compute/server.rb

```diff
@@ -237,6 +239,14 @@ def revert_snapshot(snapshot)
           end
         end
 
+        def cdroms(opts = {})
+          service.cdroms(:instance_uuid => self.id).all(opts)
+        end
```

Comment:
Yeap. it does not work with custom parameters yet.
I guess it would be a great thing but lets not worry about it now. =)

So jus remove the DSL part. =)